setup.py

```diff
@@ -67,7 +67,7 @@
     python_requires=">=3.9",
     install_requires=[
         # primary requirements for unit and property models
-        "idaes-pse >=2.7.0,<2.8.0rc0",
+        "idaes-pse @ https://github.com/watertap-org/idaes-pse/archive/2.7.0.dev.watertap.2025.01.21.zip",
```

Comment:
Need to update this

Reply:
@lbianchi-lbl reminder to check this out and ensure update to latest idaes release works here

Reply:
All the commits I just made starting with `35d51f8` were made with idaes-pse=2.8.0.dev0 and these tests passing locally.